swift/Tests/SwiftNoosphereTests/NoosphereTests.swift

```diff
@@ -517,6 +517,70 @@ final class NoosphereTests: XCTestCase {
         ns_free(noosphere)
     }
     
+    func testAuthorizeKeyAndRevokeAnAuthorization() throws {
+        let noosphere = ns_initialize("/tmp/foo", "/tmp/bar", nil, nil)
+
+        ns_key_create(noosphere, "bob", nil)
+
+        let sphere_receipt = ns_sphere_create(noosphere, "bob", nil)
+
+        let sphere_identity_ptr = ns_sphere_receipt_identity(sphere_receipt, nil)
+        let sphere_mnemonic_ptr = ns_sphere_receipt_mnemonic(sphere_receipt, nil)
+        
+        let sphere = ns_sphere_open(noosphere, sphere_identity_ptr, nil)
+        
+        let expectation = self.expectation(description: "File contents are read")
+
+        nsSphereAuthorityAuthorize(noosphere, sphere, name: "alice", did: "did:key:alice") {
+            (error, authorization_ptr) in
+            
+            if error != nil {
```

Comment:
Do these error checks actually fail the test/return non-0 if `error != nil`?  edit: ah i see the `expectation` check